src/components/input-switch/numeric-field.tsx

```diff
@@ -8,16 +8,17 @@ import { VALIDATE_VALUE_BY_TYPE } from '@isrd-isi-edu/chaise/src/utils/input-uti
 
 const NumericField = (props: InputFieldProps): JSX.Element => {
   let rules;
-  if (props.type === 'integer2' || props.type === 'integer4' || props.type === 'integer8') {
+  if (props.type === 'int') {
+    // only 'int' uses a regular expression for validation
     rules = {
-      validate: VALIDATE_VALUE_BY_TYPE[props.type]
+      pattern: VALIDATE_VALUE_BY_TYPE[props.type]
```

Comment:
I feel like at this point it's easier if we also change the int validator to be a function. And then just use the regular expression inside it. That's exactly what we're doing for other `int2`, `int4`, etc anyways. 

That way all of our validators are just functions and we don't need to worry about pattern vs. validate here.